instructions

```diff
@@ -0,0 +1,105 @@
+Install Ruby 3.3.4 on Ubuntu 22.04.4 LTS
+
+I Installed https://github.com/rbenv/rbenv as the os version is not recent
+
+I mostly followed https://shape.host/resources/step-by-step-guide-installing-ruby-on-rails-with-rbenv-on-ubuntu-22-04
+
+To start the installation process, we first need to update the package list:
+
+  sudo apt update
+  Next, we will install the dependencies required to install Ruby:
+  
+  sudo apt install git curl libssl-dev libreadline-dev zlib1g-dev autoconf bison build-essential libyaml-dev libreadline-dev libncurses5-dev libffi-dev libgdbm-dev
+  Once the dependencies are installed, we can proceed to install rbenv itself. Use the following command to fetch the install script from Github and run it:
+  
+  curl -fsSL https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-installer | bash
+  After the installation is complete, we need to add rbenv to our PATH environment variable. Modify the ~/.bashrc file to include the rbenv command line utility:
+  
+  echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+  Additionally, add the following line to the ~/.bashrc file to ensure rbenv loads automatically:
+  
+  echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+  To apply the changes made to the ~/.bashrc file, run the following command:
+  
+  source ~/.bashrc
+
+This also says how to install ruby 3.3.4
+
+To install Ruby version 3.3.4, use:
+
+  rbenv install 3.3.4
+  The installation process may take some time to complete. Once it’s done, set the installed version as the global default:
+  
+  rbenv global 3.3.4
+  To verify that Ruby is installed correctly, check its version:
+  
+  ruby -v
+
+Then install bundler gems 
+
+echo '# Install Ruby Gems to ~/gems' >> ~/.bashrc
+echo 'export GEM_HOME="$HOME/gems"' >> ~/.bashrc
+echo 'export PATH="$HOME/gems/bin:$PATH"' >> ~/.bashrc
+source ~/.bashrc
+
+gem install bundler
+
+Get MarlinDocumentation
+
+git clone https://github.com/MarlinFirmware/MarlinDocumentation.git
+
+Update Gemfile with new gems versions
```

Comment:
All the DIFF stuff can be removed from this file.

Simply deleting `Gemfile.lock` and running `bundle install` in the folder should do all the right things for an older `vendor/bundle` folder. The `Gemfile` will already be in the correct state from doing the `git pull`. If there are any issues with `bundle install` not working, deleting the `vendor/bundle` folder and then running `bundle install` again should do the trick.

The README instructions for all platforms should now have everything covered all the way from the install of Ruby to preview of documentation. Every "install" of the documentation can be treated as a "fresh install" and if savvy contributors are maintaining work branches, they already know how to `git rebase` them.